apps/console/src/features/server-configurations/components/governance-connector-grid.tsx

```diff
@@ -0,0 +1,285 @@
+                    <HexagonTwoIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.MFA_CONNECTOR_CATEGORY_ID:
+                return (
+                    <ShieldCheckIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.SESSION_MANAGEMENT_CONNECTOR_ID:
+                return (
+                    <UserDatabaseIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.ASK_PASSWORD_CONNECTOR_ID:
+                return (
+                    <UserGearIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.ALTERNATIVE_LOGIN_IDENTIFIER:
+                return (
+                    <ArrowLoopRightUserIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.MULTI_ATTRIBUTE_LOGIN_CONNECTOR_ID:
+                return (
+                    <UserDocumentIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.USERNAME_VALIDATION:
+                return (
+                    <GenericIcon
+                        size="micro"
+                        icon={ getSettingsSectionIcons().usernameValidation }
+                        transparent
+                        shape={ "square" }
+                    />
+                );
+            case ServerConfigurationsConstants.SELF_SIGN_UP_CONNECTOR_ID:
+                return (
+                    <UserPlusIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.LOGIN_ATTEMPT_SECURITY:
+                return (
+                    <HexagonTwoIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.PASSWORD_RECOVERY:
+                return (
+                    <ShieldUserPencilIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.USERNAME_RECOVERY:
+                return (
+                    <EnvelopeAtIcon className="icon" />
+                );
+            default:
+                return <GearIcon fill="primary" className="icon" />;
+        }
+    };
+
+    return (
+        <div>
+            { connectorCategories?.map((category: any, index: number) => {
+                return (
+                    <div className="catergory-container" key={ index }>
+                        <Typography 
+                            color="text.primary"
+                            className="mb-3" 
+                            variant="h4"
+                        >
+                            { category?.title }
+                        </Typography>
+                        <div className="governance-connector-list-grid-wrapper" data-componentid={ componentId }>
+                            <div className="governance-connector-list-grid">
+                                {
+                                    category.connectors.map((connector: any) => {
```

Comment:
Let's fix the other places as well.

```suggestion
                                    category?.connectors?.map((connector: any) => {
```